xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-theme/xwiki-platform-flamingo-theme-bootswatch/src/main/resources/FlamingoThemes/Paper.xml

```diff
@@ -2400,14 +2400,14 @@ input[type="checkbox"],
 div.main {
   .box-shadow(0 1px 4px rgba(0,0,0,.3));
   border: medium none;
-  border-radius: 2px;
+  border-radius: 3px;
   margin-top: 20px;
 }
 
 .panel-width-Small div.panel.Applications{
   .box-shadow(0 1px 4px rgba(0,0,0,.3));
   border: medium none;
-  border-radius: 2px;
+  border-radius: 3px;
```

Comment:
Since Color themes can decide their own border radiuses in the advanced section (or with variables), why are we changing the color themes here?

Reply:
@lucaa Thanks. To answer your comments, I'll revert the changes in `FlamingoThemes` and replace the less variables I've missed.
Regarding the remainder of the hardcoded radius, do you prefer to keep my changes, or to revert it (i.e., do you rather have them with a "too small" radius by default, or "too large" one)?

Reply:
"too large" for the case when someone changes the defaults should be fine, so you can leave the hardcoded values aligned with the new default radiuses.